### PR TITLE
move devcommands to northstar.customservers

### PR DESCRIPTION
**Pull request by BobTheBob9**

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
Uh, my Squirrel knowledge is still non-existent. Probably better if someone else takes a look :P

File: Northstar.Custom/mod.json [hunk 1]
```diff
@@ -416,13 +409,7 @@
 			}
 		}
 	],
-	"Maps": [
-		{
-			"Name": "mp_skyway_v1",
-			"VPK": "englishclient_mp_skyway_v1",
-			"RPak": "sp_skyway_v1"
-		}
```
Comment (emma-miler) [inline Northstar.Custom/mod.json:424]: What's up with this being removed?

**Comment by GeckoEidechse** [comment]
Testing locally by launching client server, enabling `sv_cheats` and no-clipping didn't crash my game so I guess it works? :P

**Comment by emma-miler** [comment]
Fine by me then

**Review by emma-miler — APPROVED** [review]

File: Northstar.Custom/mod.json (hunk 1 above)
Comment (BobTheBob9) [reply Northstar.Custom/mod.json:424]: because i put it there for testing like ages ago and it shouldn't be there

Comment (BobTheBob9) [reply Northstar.Custom/mod.json:424]: like it literally does nothing, mod format doesn't support those keys

Comment (emma-miler) [reply Northstar.Custom/mod.json:424]: lol aight

**Pull request merged**